### PR TITLE
gcc: Fix "Enabling language "jit" requires --enable-host-shared".

### PR DESCRIPTION
**Pull request by SaberUK**
Exactly what it says on the tin.

### CODE REVIEW

File: Library/Formula/gcc.rb [hunk 1]
```diff
@@ -137,7 +137,7 @@ def install
       args << "--enable-multilib"
     end
 
-    args << "--enable-host-shared" if build.with? "jit"
+    args << "--enable-host-shared" if build.with? "jit" or build.with? "all-languages"
```
Comment (DomT4) [inline Library/Formula/gcc.rb]: ` || ` rather than ` or `.

Comment (SaberUK) [inline Library/Formula/gcc.rb]: Changed.

**Comment by SaberUK** [comment]
The build error on Mountain Lion seems to be a Jenkins issue:

    ==> brew bottle --rb gcc						Build timed out (after 60 minutes).
    Marking the build as aborted.
    Build was aborted

**Comment by DomT4** [comment]
Yeah, we set the build timeouts at 1 hour to stop any one build taking up too many resources. The build limit will need bumping to about 1hr30 to pass ML.

Jack or Mike will bump it later if they need to. Given we don't test non-default options, the build success isn't super-important though.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/e43b8040439fbb076ddfe40fd2e5512dc8ea31fc. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:. Tiny nit: commit messages should be lower case.

**Pull request closed without merging**